src/modules/announcements/components/AnnouncementsList.jsx

```diff
@@ -0,0 +1,53 @@
+import ReactStars from "react-rating-stars-component";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import styles from "./AnnouncementsList.module.scss";
+
+export const AnnouncementsList = ({ filteredAnnouncements }) => {
+  if (filteredAnnouncements.length === 0) {
+    return (
+      <h3 className={styles.announcementNotFound}>
+        Przeprazamy, niestety nie ma żadnych ogłoszeń spełniających kryteria
+        Twoich wyszukiwań
+      </h3>
+    );
+  }
+  return (
+    <div className={styles.allAnnouncements}>
+      <ul>
+        {filteredAnnouncements.map((e) => (
+          <li key={e.id}>
+            <Link to={`/announcements/ ${+e.id}`}>
```

Comment:
Same like on `CRA-18` move it before component to `ANNOUCEMENT_ROUTE` and use a string literal here